vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -206,9 +206,13 @@ def process_weights_after_loading(self, layer: Module) -> None:
         # If checkpoint is fp8, handle that there are N scales for N
         # shards in a fused module
         else:
+            layer.weight_scale.data[layer.weight_scale.data == torch.finfo(
+                torch.float32).min] = 1
```

Comment:
Could you pull out `torch.finfo(torch.float32).min` into a constant in this file so we can use a single reference when also using it in `create_weights()`? i.e. `UNINITIALIZED_SCALE = torch.finfo(torch.float32).min`

What is the case that this will be happening? This seems like it may cover up failed weight loading, where we might want to raise an exception.

Reply:
> Could you pull out torch.finfo(torch.float32).min into a constant in this file 
Will do
This is for the case when a otherwise quantize model has unquantized layers. These layers will get converted to fp8 with the scale of 1.0, but this scale will not get loaded, as it is not in the model

Reply:
I see, in that case then I think this is an incorrect fix. We have infrastructure for referencing the module name against the ignore list in the quantization config. We should fix the issue if they name is not matching or detected from the ignore list here: https://github.com/vllm-project/vllm/blob/622b7ab955186f37879208d7a30e9faf985be220/vllm/model_executor/layers/quantization/fp8.py#L85-L88

Reply:
That's good to know, thanks. In this case we only need the kv remap and 3d tensor changes. I will remove this one